omnichain/nft/contracts/NFT.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.7;
+
+import "@zetachain/protocol-contracts/contracts/zevm/SystemContract.sol";
+import "@zetachain/protocol-contracts/contracts/zevm/interfaces/zContract.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
```

Comment:
Counters was removed on OZ version 5. Try not to use it or tag explicitly the OZ version, otherwise, the code will no work for example on remix.

Reply:
93e65d6